.eslintignore

```diff
@@ -7,3 +7,5 @@
 CHANGELOG.md
 dist
 node_modules
+/test/regressions/build
+/test/regressions/screenshots
```

Comment:
```suggestion
```

There are no JavaScript file in this folder.

Reply:
agree but I also needed to unlock myself